src/index.js

```diff
@@ -30,6 +64,7 @@ const store = {
   },
 
   messageQueue: [],
+  history: chatHistory.get(),
```

Comment:
let this be undefined since you are reading history once session connects anyway